lib/core/formats.js

```diff
@@ -0,0 +1,49 @@
+const { logger } = require('../../config/logger');
+
+const FormatTypes = {
+  UPPERCASE_FORMAT: 'uppercase',
+  LOWERCASE_FORMAT: 'lowercase',
```

Comment:
```suggestion
  LOWERCASE: 'lowercase',
```

We are already in a FormatType Object, I don't believe we should add "Format" at the end of every key.
As in JavaScript it is toLowerCase I think "LOWERCASE" or "UPPERCASE" is enough

Reply:
👍